devicetypes/smartthings/orvibo-Moisture-Sensor.src/orvibo-Moisture-Sensor.groovy

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ *
+ *  Orvibo Moisture Sensor
+ *
+ *  Author: Deng Biaoyi/biaoyi.deng@samsung.com
+ *
+ *  Date:2018-07-03
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "Orvibo Moisture Sensor", namespace: "smartthings", author: "SmartThings", vid: "generic-leak", mnmn:"SmartThings", ocfDeviceType: "x.com.st.d.sensor.moisture") {
+		capability "Configuration"
+		capability "Refresh"
+		capability "Water Sensor"
+		capability "Sensor"
+		capability "Health Check"
+		capability "battery"
```

Comment:
As a general style rule, we capitalize capability names.

Reply:
Ok, I have fixed it, please review again.